trace/src/main/org/jetbrains/lincheck/trace/Deserialization.kt

```diff
@@ -257,7 +266,7 @@ class LazyTraceReader(
         loadTracePoints(
             threadId = parent.threadId,
             maxRead = Integer.MAX_VALUE,
-            reader = this::readTracePointWithChildAddresses,
+            reader = { readTracePointWithPostprocessor(this::readTracePointWithChildAddresses) },
```

Comment:
BTW, why not just pass it as `this::readTracePointWithPostprocessor` and remove lambda argument from `readTracePointWithPostprocessor` (it looks like you always pass the same read callback anyway).